NOTICE

```diff
@@ -0,0 +1,14 @@
+   GSoC '21 Proxy-WASM filters - 3scale labs
+   Copyright (c) 2020, 2021 Red Hat, Inc.
```

Comment:
There's no copyright attribution required for 3scale/Red Hat unless you sign such a thing - in other words, you can (and should) use your own copyright :)

Reply:
fixed